sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/PropertyConfiguratorTest.java

```diff
@@ -364,6 +381,50 @@ public void trackEventWithEmptyProperties() {
         assertEquals(Collections.emptyList(), log.getTypedProperties());
     }
 
+
+    @Test
+    public void trackEventWithCommonTypedProperties() {
+
+        /* Create transmission target. */
+        AnalyticsTransmissionTarget target = Analytics.getTransmissionTarget("test");
+
+        /* Set common properties for various types. Some of which are invalid. */
+        target.getPropertyConfigurator().setEventProperty("myString", "hello");
+        target.getPropertyConfigurator().setEventProperty("myNullString", (String) null);
+        target.getPropertyConfigurator().setEventProperty("myTrue", true);
+        target.getPropertyConfigurator().setEventProperty("myFalse", false);
+        target.getPropertyConfigurator().setEventProperty("myLong", 123);
+        target.getPropertyConfigurator().setEventProperty("myDate", new Date(456));
+        target.getPropertyConfigurator().setEventProperty("myNullDate", (Date) null);
+        target.getPropertyConfigurator().setEventProperty("myDouble", -3.14);
```

Comment:
Add a case for scientific notation of double like "1.2345e" would be great

Reply:
Another method has a simple double so will just change this one